js/data.js

```diff
@@ -1059,7 +1069,7 @@ var data = {
         dozent: ["Dr. Anselm Brandi-Dohrn", "Monika Menz"],
         nameLV: "IT-Recht"
     },
-    komplexitaestheorie: {
+    komplexitaetstheorie: {
```

Comment:
Please do not change the keys. This prevents the localstorage from working, and breaks existing course plans.
I see the spelling error, but we have to live with it now.